pkg/istio/rbac/v2/provider.go

```diff
@@ -78,6 +83,17 @@ func (p *v2) ConvertAthenzModelIntoIstioRbac(athenzModel athenz.Model, serviceNa
 			if err != nil {
 				continue
 			}
+
+			// Drop the query parameters from the HTTP path in the assertions due to the difference
+			// in the RBAC Envoy permissions config created by Authentication Policy and ServiceRole/ServiceRoleBindings.
```

Comment:
This should be authorization policy as authentication policy is a different resource used for mTLS etc.

Reply:
Ah, ya. Thanks for catching that